### PR TITLE
feat(builtins): impl srv.info, dist.get_blueprint

### PR DESCRIPTION
**Pull request by folex**

### CODE REVIEW

File: particle-services/src/app_services.rs [hunk 1]
```diff
@@ -187,6 +187,30 @@ impl ParticleAppServices {
         Ok(service_id)
     }
 
+    pub fn get_service_info(
+        &self,
+        worker_id: PeerId,
```
[the remark this answers is not in the document]
Comment (folex) [reply particle-services/src/app_services.rs:192]: we call it `worker_id` everywhere, and there's no other id except peer id

**Review by justprosh — APPROVED** [review]

File: particle-services/src/app_services.rs (hunk 1 above)
[the remark this answers is not in the document]
Comment (kmd-fl) [reply particle-services/src/app_services.rs:192]: What's the difference between this `worker_id` and `service.worker_id`? I though `service.worker_id` is a service/spell id of a worker, no?

[the remark this answers is not in the document]
Comment (justprosh) [reply particle-services/src/app_services.rs:192]: There is no difference. service.worker_id indicates where is this service deployed

[the remark this answers is not in the document]
Comment (folex) [reply particle-services/src/app_services.rs:192]: it's the same thing. Services now can belong to `worker_id`, aka `virtual peer`.

[the remark this answers is not in the document]
Comment (kmd-fl) [reply particle-services/src/app_services.rs:192]: Why do we start using the word "worker" (which isn't descriptive at all) and not "virtual peer" (which is self-explanatory) then?

**Pull request merged**